### ISSUE TITLE
Update to 1.9.2!

### ISSUE THREAD

**Issue by INDRAPhilip** [RetroScaler/Blueretro-PS#4]
New version corrects some errors and finally adds the button pressure option for all main controls, which in this case are PS3 controls! Can you add other controls too?

Bug fixes
Fix polarity of the RY axis for generic wireless devices (Thanks @hworld)
Fix mapping regression for some 8bitdo controller
Fix sign of 8 bits logical min & max in HID parser

New features
Add support for PS2/3 pressure buttons

**Comment by mhmetalheart** [comment]
agree, please update to 1.9.2.

**Comment by SavageCore** [comment]
Just follow the standard OTA update instructions: https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#5---ota-fw-update-page

**Comment by arnauos** [comment]
> Just follow the standard OTA update instructions: https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#5---ota-fw-update-page
> 
> The releases from itch work fine.

That is correct, the fact they provide a bin file and a software to flash it may lead to confusion.

**Comment by Metro-City** [comment]
Blueretro Neo Geo? i've problem to sync with 2nd adapter, together doesn't work always blink red.

**Comment by RetroScaler** [comment]
Hi Metro-City, BIG THANKS for your purchase and really sorry for you suffering, pls you can join us on discord, discord.gg/aryJd43VWJ <https://t.co/w9P7Wy0Q4U >, a better place to talk and address to your issue 
------------------------------------------------------------------
发件人：Metro-City ***@***.***>
发送时间：2024年6月27日(星期四) 22:15
***@***.***>
***@***.***>
主　题：Re: [RetroScaler/Blueretro-PS] Update to 1.9.2! (Issue #4)
Blueretro Neo Geo? i've problem to sync with 2nd adapter, together doesn't work always blink red.
